ytfzf

```diff
@@ -468,10 +465,16 @@ scrape_channel () {
 	]')"
 	
 	videos_data="$(get_video_data "$videos_json")"
+	#if select all or random select keep all the links so that they can all be selected
+	[ $select_all -eq 0 ] && videos_data="$(echo "$videos_data" | head -${link_count})"
+	if [ $auto_select -eq 1 -o $random_select -eq 1 ]; then
+	    user_selection
+	    echo "$urls" >> "$tmp_video_urls_file"
+	fi
```

Comment:
This seems to work,
I would be much cleaner if we did our usual auto-select, random select, directly on the video data. And push it through User selection with `-A`

Reply:
idk how to make it cleaner, and keep the functionality of having it play from each subscription